pkg/dock/proto/dock.proto

```diff
@@ -112,6 +112,8 @@ message CreateVolumeOpts {
     map<string, string> replicationDriverData = 13;
     // The uuid of replication
     string replicationId = 14;
+    // The size of snapshot
+    int64 snapshotSize = 15;
```

Comment:
See my previous comment.  It seems not necessary to add this snapshotSize in CreateVolumeOpts.  We can compare snapshot size with volume size earlier and bail out if it is not right.